client/src/components/filters.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div>
+    <v-card class="ma-5 pa-2 border rounded ma-2 align-self-start">
+      <v-row class="ma-5">
+        <v-col cols="12" class="py-5 px-2">
+          <span>You can change the selected office to discover the team in other offices.
+          </span>
+        </v-col>
+      </v-row>
+      <v-row class="ma-5">
+        <v-col cols="12" sm="12" md="2" class="px-2 align-self-start">
+
+          <div>
+            <h3> Select Office</h3>
+          </div>
+        </v-col>
+        <v-col cols="12" sm="12" md="10" class="pa-2 align-self-start">
+          <div>
+
+
+
+            <v-autocomplete v-model="country" :items="countries" label="Office" @update:model-value="handleCountryChange"
+              return-object item-title="country">
+            </v-autocomplete>
```

Comment:
If the label is`Office` why we named the variable `country/countries`?
```suggestion
            <v-autocomplete v-model="office" :items="offices" label="Office" @update:model-value="handleCountryChange"
              return-object item-title="country">
            </v-autocomplete>
```